### ISSUE TITLE
Plugin not updating via Updater since moved from BitBucket to GitHub

### ISSUE THREAD

**Issue by yaronguez** [afragen/github-updater#674]
Hi I moved a plugin from a private BitBucket repo to a public GitHub repo. Since then the plugin does not update through the WordPress Dashboard either by the plugins page or the Updates page. In both cases the new version is detected, the offer to upgrade is presented, and the upgrade performs with no errors, stating that the plugin update was successful. However after refreshing the old plugin version is still there.

I've tried:
Clearing the GitHub Updater Cache and WP Engine Cache
Completely removing the plugin and re-installing it via GitHub Updater. The latest version gets installed but no future updates succeed the upgrade process.

Any ideas? 

This is the repo:
https://github.com/crypteron/encryptwp

**Comment by afragen** [comment]
Hey @yaronguez! The problem is that your release does not match your latest version. The way I have set up GitHub Updater is that if you start tagging releases you must continue. What happens is GHU detects a version bump in the main plugin file but uses the latest tagged release to update.

Does that make sense?

**Comment by yaronguez** [comment]
Hi @afragen , thanks for such a fast turn around! I had actually tried creating a new tagged release that matched the latest version as part of my testing from 1.1.2 to 1.1.3 and it didn't seem to make a difference. That's when I manually re-installed 1.1.3.

I just tried again, though, this time unchecking the "pre-release" option for 1.1.4 and it now works. Is pre-release not allowed?

**Comment by afragen** [comment]
A pre-release is still a tagged version. GitHub API doesn’t make a distinction when getting the list of tags. Sorry.

What I saw in your repository, before the latest tag, was 2 tags (1.1.2 and 1.1.3). What would normally happen if the latest version is greater than 1.1.3 is any update would use the latest tag and you would be in a cycle where you would _see_ and update, but the update would only re-install the last tag.

**Comment by yaronguez** [comment]
Interesting. Thanks for the clarification and keep up the good work!

**Comment by afragen** [comment]
Anytime and thanks.

**Comment by ensignrolaren** [comment]
I had the same experience and it's good to know that if you start tagging releases you must continue. I only started tagging because I was trying to figure out why I was in a failed update loop (but it was because I'm new to implementing git and forgot to sync my changes). Now I'm stuck making a new tagged release every single time and it's driving me bonkers and way more effort than my use case calls for. Any way to undo it? I've already tried to delete all my tags to see if I can just stage & push my changes without creating a new release, but I keep getting in the failed update loop that the above poster described still.

**Comment by afragen** [comment]
If you delete all of your tags it should work. 

What is the repo URI?

**Comment by ensignrolaren** [comment]
It's a private repo sadly or I'd link it. But hopefully this means I just did it wrong. I will try again to delete all the tags and see if I get it straightened. Thanks much!

**Comment by afragen** [comment]
If you still have problems you can DM me in the Git Updater slack and we can figure it out.

**Comment by ensignrolaren** [comment]
Will keep that in mind and thank you so much!

**Comment by ensignrolaren** [comment]
That worked perfectly thanks! I didn't realize before that I had only deleted the releases and not the tags. I needed to use the command line since the github web UI doesn't provide a method to delete tags. For posterity: I [found this article] (https://www.abeautifulsite.net/posts/how-to-delete-a-tag-on-github/) which gave the following commands to delete the tags:
```
git tag -d [tag];
git push origin :[tag]
```
Replacing [tag] with the name of your tag.

Thanks again!
